avocado/core/html.py

```diff
@@ -130,19 +130,20 @@ def tests(self):
                    "INTERRUPTED": "danger"}
         test_info = []
         results_dir = self.results_dir(False)
-        for t in self.result.tests:
-            formatted = {}
-            logdir = os.path.join(results_dir, 'test-results', t['logdir'])
-            formatted['logdir'] = os.path.relpath(logdir, self.html_output_dir)
+        for tst in self.result.tests:
+            tst = tst.copy()    # we don't want to override other's results
```

Comment:
Previous comment:

--

This change the `result.tests` entries, which then break expecations of other parts of the code.  One example is:

```
  File "/home/travis/build/avocado-framework/avocado/avocado/core/html.py", line 142, in tests

    time.localtime(tst['time_start']))

TypeError: a float is required
```

That's why I created the `formatted` dict.  But yeah, it misses some of the items, and the failure is a silent field on the generated HTML.

The way to go here is to keep `formatted` and add the missing fields.

--

But yeah, copying is not that bad.